package.json

```diff
@@ -32,7 +32,8 @@
     "greenkeeper[bot] <23040076+greenkeeper[bot]@users.noreply.github.com>",
     "greenkeeper[bot] <greenkeeper[bot]@users.noreply.github.com>",
     "kumavis <aaron@kumavis.me>",
-    "kumavis <kumavis@users.noreply.github.com>"
+    "kumavis <kumavis@users.noreply.github.com>",
+    "Chris Hafey <chafey@gmail.com>"
```

Comment:
Just a note for next time: you don't need to add your name to this list manually as this list of contributors is updated automatically on a release.